ast/node.go

```diff
@@ -526,6 +526,15 @@ type LambdaExpr struct {
 
 func (i *LambdaExpr) Pos() Pos { return i.Lcurlybrace }
 
+// ParenExpr node represents a parenthesized expression.
+// vimlparsr: PARENEXPR .value
+type ParenExpr struct {
+	Lparen Pos  // position of "("
+	X      Expr // parenthesized expression
+}
+
+func (i *ParenExpr) Pos() Pos { return i.Lparen }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method ParenExpr.Pos should have comment or be unexported